tests/eas/acceptance.py

```diff
@@ -378,193 +378,65 @@ class WakeMigration(unittest.TestCase):
     the big cpus when they are big.
     """
 
-    @classmethod
-    def setUpClass(cls):
-        cls.params = {}
-        cls.env = TestEnv(test_conf=TEST_CONF)
-        cls.task_prefix = "wmig"
-        cls.trace_file = os.path.join(cls.env.res_dir, "wake_migration.dat")
-        cls.log_file = os.path.join(cls.env.res_dir, "wake_migration.json")
-        cls.populate_params()
-        cls.tasks = cls.params.keys()
-        cls.num_tasks = len(cls.tasks)
-        local_setup(cls.env)
-        cls.run_workload()
-        cls.s_assert = SchedMultiAssert(
-            cls.trace_file,
-            cls.env.topology,
-            execnames=cls.tasks)
-        cls.offset = cls.get_offset(cls.tasks[0])
-        cls.log_fh = open(os.path.join(cls.env.res_dir, cls.log_file), "w")
-
-    @classmethod
-    def tearDownClass(cls):
-        cls.log_fh.close()
-
-    @classmethod
-    def populate_params(cls):
-        num_big_cpus = len(cls.env.target.bl.bigs)
-
-        for i in range(num_big_cpus):
-            task_name = "{}_{}".format(cls.task_prefix, i)
-            cls.params[task_name] = Step(**STEP_WORKLOAD).get()
-
-        cls.phase_duration = STEP_WORKLOAD["time_s"]
-
-    @classmethod
-    def run_workload(cls):
-        wload = RTA(
-            cls.env.target,
-            "wake_migration",
-            calibration=cls.env.calibration())
-        wload.conf(kind="profile", params=cls.params)
-        cls.env.ftrace.start()
-        wload.run(
-            out_dir=cls.env.res_dir,
-            background=False)
-        cls.env.ftrace.stop()
-        trace = cls.env.ftrace.get_trace(cls.trace_file)
+    conf_basename = "wake_migration.config"
+    # TODO...
+    phase_duration = WORKLOAD_DURATION_S
+
+    def _assert_switch(self, experiment, expected_switch_to, phases):
+        if expected_switch_to == "big":
+            switch_from = self.te.target.bl.littles
+            switch_to   = self.te.target.bl.bigs
+        elif expected_switch_to == "little":
+            switch_from = self.te.target.bl.bigs
+            switch_to   = self.te.target.bl.littles
+        else:
+            raise Exception("Invalid expected_switch_to")
+
+        expected_time = (self.get_start_time(experiment) +
+                         WORKLOAD_DURATION_S * phases)
```

Comment:
I know we are almost never consistent with this part of pep8, but it recommends breaking line before the `+` and removing the spaces around the `*` because it takes higher preference

``` python
        expected_time = (self.get_start_time(experiment)
                         + WORKLOAD_DURATION_S*phases)
```

https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator
https://www.python.org/dev/peps/pep-0008/#other-recommendations